### PR TITLE
Tdarr v2 templates for Server and Node

### PR DESCRIPTION
**Pull request by HaveAGitGat**

### CODE REVIEW

File: templates/tdarr_node.xml [hunk 1]
```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0"?>
+<Container version="2">
+  <Name>tdarr</Name>
```
Comment (Roxedus) [inline templates/tdarr_node.xml:3]: Probably worth renaming this to something exclusive

Comment (HaveAGitGat) [reply templates/tdarr_node.xml]: Thought I'd changed that, good catch thanks. Updated.

File: templates/tdarr.xml [hunk 2]
```diff
@@ -3,17 +3,19 @@
   <Name>tdarr</Name>
   <Repository>haveagitgat/tdarr</Repository>
   <Registry>https://hub.docker.com/r/haveagitgat/tdarr/</Registry>
-  <Network>bridge</Network>
+  <Network>host</Network>
```
Comment (Roxedus) [inline templates/tdarr.xml:6]: Any real reason its in host? just catched that

Comment (HaveAGitGat) [reply templates/tdarr.xml:6]: Just makes it a bit easier to setup as with network bridge you have to expose the ports explicitly.

Comment (Roxedus) [reply templates/tdarr.xml:6]: But you already have set up those port in the template? I would like to avoid host networking as long as possible.

Comment (HaveAGitGat) [reply templates/tdarr.xml]: Ok switched it to bridge.

**Comment by Roxedus** [comment]
And rebase to testing

**Comment by HaveAGitGat** [comment]
Rebased onto testing :)

**Pull request merged**